.devcontainer/Dockerfile

```diff
@@ -83,6 +101,25 @@ RUN echo "source /entrypoint.sh" >> /home/vscode/.bashrc
 # Install the Jupyter kernel
 RUN . /entrypoint.sh && \
        python3 -m ipykernel install \
-        --name py311-exago\
-        --display-name ExaGO \
+        --name py311-exago \
+        --display-name "ExaGO" \
         --prefix=
+
+# Install the mpi4py Jupyter kernel
+RUN . /entrypoint.sh && \
+       python3 -m ipykernel install \
+        --name py311-mpi4py-exago \
+        --display-name "ExaGO w/ MPI" \
+        --prefix=
+
+# Make modifications that are necessary to run mpi4py in kernelspec
+RUN sed -i 's/\"\/opt\/views\/view\/bin\/python3\"/\"mpiexec\", \"-n\", \"1\", \"\/opt\/views\/view\/bin\/python3\"/g' /usr/local/share/jupyter/kernels/py311-mpi4py-exago/kernel.json
```

Comment:
I assume this doesn't affect regular notebooks that don't use mpi - it just allows mpi4py to work if used?

Reply:
Intention is that there are two kernels available. One for running with MPI, and one without. 

This edits the MPI named kernel to actually use MPI. This is impossible to do without editing the file directly as far as I can tell.

I also wonder if it isn't necessary, since I tried that going down a rabbit hole